operators/BaseOperator.js

```diff
@@ -122,7 +127,7 @@ class BaseOperator {
           });
       } else {
         processingLockStatus.conflict = true;
-        logger.info(`Request with options ${JSON.stringify(options)} is picked by process with ip ${lockedByManager} at ${processingStartedAt}`);
+        logger.debug(`Resource ${objectBody.metadata.name}  ${resourceGroup}, ${resourceType}, ${queryString} - is picked by process with ip ${lockedByManager} at ${processingStartedAt}`);
```

Comment:
Generally if you open up the logs we see lot of such stmts which really dont add value since the watchers refresh at refresh interval and logs really get bloated with lots of such stmts which add no value per se. (i.e. we see xxx picked up by other host msgs a lot). So based on my experience I lowered it.

Reply:
Sure, just thought that this might help in debugging which process picked up the request.